docker/bashrc.sh

```diff
@@ -6,5 +6,12 @@ if [ "$BUILD_ENV" = "dev" ]; then
     alias djrun="python manage.py runserver 0.0.0.0:8000"
     alias djrunplus="python manage.py runserver_plus 0.0.0.0:8000"
     alias djtest="python manage.py test --settings=ons_alpha.settings.test"
-    alias honcho="honcho -f docker/Procfile"
 fi
+
+# nvm
+if [ -a "$HOME/.nvm/nvm.sh" ]; then
+  export NVM_DIR="$HOME/.nvm"
+  source "$NVM_DIR/nvm.sh"
+fi
+
+alias honcho="honcho -f docker/Procfile"
```

Comment:
Nit: I think it makes sense to keep these files inside the above `if` - they're only relevant for dev.

Reply:
I'd actually meant this and the `nvm` sourcing - since `nvm` will only be installed in dev anyway.